packages/typescript/src/create-package.ts

```diff
@@ -54,28 +58,81 @@ function createSymbols(
     }
 
     for (const exportedName of symbols.named ?? []) {
-      const key = keys[exportedName];
-      const sym = createTSSymbol({
-        name: exportedName,
-        scope: moduleScope,
-        refkey: key,
-        export: true,
-        default: false,
-      });
-      moduleScope.exportedSymbols.set(key, sym);
+      if (typeof exportedName === "object") {
```

Comment:
purposely split this without refactoring to show the new code path for discussion